build/yaml/crd/nsx.vmware.com_nsxserviceaccounts.yaml

```diff
@@ -0,0 +1,107 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
+  creationTimestamp: null
+  name: nsxserviceaccounts.nsx.vmware.com
+spec:
+  group: nsx.vmware.com
+  names:
+    kind: NSXServiceAccount
+    listKind: NSXServiceAccountList
+    plural: nsxserviceaccounts
+    singular: nsxserviceaccount
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: NSXServiceAccount is the Schema for the nsxserviceaccounts API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: NSXServiceAccountSpec defines the desired state of NSXServiceAccount
+            properties:
+              vpcName:
+                type: string
+            type: object
+          status:
+            description: NSXServiceAccountStatus defines the observed state of NSXServiceAccount
+            properties:
+              clusterID:
+                type: string
+              clusterName:
+                type: string
+              nsxManagers:
+                items:
+                  type: string
```

Comment:
should it not be ip other than string?

Reply:
This is generated code.
@zhengxiexie Do you know how we can generate this property as type "ip"?

Reply:
Yes, this item can be hostname or IP.
I added validation for another property whose format should be ip.